packages/access-client/test/stores/store-indexeddb.browser.test.js

```diff
@@ -12,8 +12,11 @@ describe('IndexedDB store', () => {
     // principal private key is not extractable
     const archive = data.principal.toArchive()
     assert(!(archive instanceof Uint8Array))
-    assert(archive.key instanceof CryptoKey)
-    assert.equal(archive.key.extractable, false)
+    const keys = [...Object.entries(archive.keys)]
+    assert.equal(keys.length, 1)
+    const firstKey = keys[0][1]
+    assert(firstKey instanceof CryptoKey)
+    assert.equal(firstKey.extractable, false)
```

Comment:
```suggestion
    const [[did, key], ...keys] = [...Object.entries(archive.keys)]
    assert.deepEqual(keys, [])
    assert(key instanceof CryptoKey)
    assert.equal(key.extractable, false)
```

I think this is easier to follow

Reply:
I think it would throw an unexpected error if keys was length 0, whereas the way I did it will throw an AssertionError. either is fine w/ me though.